src/class.cscf_contact.php

```diff
@@ -46,6 +47,11 @@ function __construct() {
 						case 'message':
 							$this->Message = sanitize_textarea_field( $value );
 							break;
+						case 'phone-number':
+							if ( cscf_PluginSettings::PhoneNumber() ) {
```

Comment:
No point in this if statement, as it is just sanitizing the $_POST if is exists anyway

Reply:
I simply applied the same pattern that was used for the consent message already, but probably you are right. There is no point in having the additional if-statement here.